service/worker/fx.go

```diff
@@ -62,6 +63,7 @@ var Module = fx.Options(
 	deletenamespace.Module,
 	scheduler.Module,
 	batcher.Module,
+	versioning.Module,
```

Comment:
Change the package name back to `deployment` with separate workflow files for deployment and deploymentName (same for activities)